.eslintrc.yaml

```diff
@@ -64,10 +64,12 @@ rules:
     lines-between-class-members: off # be more lenient on member declarations
     max-classes-per-file: off # helper classes are common
     max-len: [warn, 150] # more lenient on max length per line
-    new-cap: off # some class still doesn't follow this yet
-    no-underscore-dangle: off # allow underscores
+    new-cap: error
+    no-underscore-dangle: [warn, {
+      allowFunctionParams: true # designate the unused param
```

Comment:
but I don't know about this though. I think the default behavior(just remove the `off` rule) would suffice.

Reply:
For unused parameter, tsc will report error unless you prefixed it with an underscore.